### PR TITLE
Mention additional Lightning Media integrations in README

### PR DESCRIPTION
**Pull request by dakkusingh**
adding the following integrations: Imgur, Flickr, 500px

### CODE REVIEW

**Comment by phenaproxima** [comment]
I'm not against adding these modules to our README. I think it would be helpful information for people who want support for these media types in Lightning. @balsama, what do you think?

**Comment by phenaproxima** [comment]
This looks good. I'd like to change the verbiage a bit, though.

Can you put the external integrations in a separate list, with a sentence or two preceding it, explaining that these are additional third-party integration packages that Lightning does not officially maintain? While we are happy to mention that these modules exist, we are not going to be officially responsible for maintaining them, and I don't want to give people that impression.

File: README.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ The current version of media includes the following functionality:
   supported:
   * Tweets
   * Instagram posts
+  * [Facebook posts](https://www.drupal.org/project/lightning_media_facebook)
+  * [Imgur posts](https://www.drupal.org/project/lightning_media_imgur)
+  * [Flickr posts](https://www.drupal.org/project/lightning_media_flickr)
+  * [500px posts](https://www.drupal.org/project/lightning_media_d500px)
+  * [Soundcloud posts](https://www.drupal.org/project/lightning_media_soundcloud)
```
Comment (phenaproxima) [inline README.md]: "Soundcloud posts" should simply be "SoundCloud".

File: README.md [hunk 2]
```diff
@@ -43,6 +43,13 @@ The current version of media includes the following functionality:
   supported:
   * Tweets
   * Instagram posts
+  * [Facebook posts](https://www.drupal.org/project/lightning_media_facebook)
+  * [Imgur posts](https://www.drupal.org/project/lightning_media_imgur)
```
Comment (phenaproxima) [inline README.md]: Can this just say "Imgur"?

File: README.md [hunk 3]
```diff
@@ -43,6 +43,13 @@ The current version of media includes the following functionality:
...
+  * [Flickr posts](https://www.drupal.org/project/lightning_media_flickr)
```
Comment (phenaproxima) [inline README.md]: Can this just be "Flickr"?

File: README.md [hunk 4]
```diff
@@ -43,6 +43,13 @@ The current version of media includes the following functionality:
...
+  * [500px posts](https://www.drupal.org/project/lightning_media_d500px)
+  * [Soundcloud posts](https://www.drupal.org/project/lightning_media_soundcloud)
+  * [Tumblr posts](https://www.drupal.org/project/lightning_media_tumblr)
+  * [Spotify posts](https://www.drupal.org/project/lightning_media_spotify)
```
Comment (phenaproxima) [inline README.md]: Can this be just "Spotify"?

File: README.md [hunk 5]
```diff
@@ -43,6 +43,13 @@ The current version of media includes the following functionality:
   supported:
   * Tweets
   * Instagram posts
+  * [Facebook posts](https://www.drupal.org/project/lightning_media_facebook)
+  * [Imgur posts](https://www.drupal.org/project/lightning_media_imgur)
+  * [Flickr posts](https://www.drupal.org/project/lightning_media_flickr)
+  * [500px posts](https://www.drupal.org/project/lightning_media_d500px)
+  * [Soundcloud posts](https://www.drupal.org/project/lightning_media_soundcloud)
+  * [Tumblr posts](https://www.drupal.org/project/lightning_media_tumblr)
```
Comment (phenaproxima) [inline README.md]: Let's remove the word "posts" and simply say "Tumblr".

**Review by phenaproxima — CHANGES_REQUESTED** [review]
Getting there!

**Comment by dakkusingh** [comment]
Let me make the above changes :)
thanks for the feedback @phenaproxima

**Review by phenaproxima — APPROVED** [review]

**Comment by dakkusingh** [comment]
@phenaproxima I think we might be there now :)
please can you take a look?

File: README.md [hunk 6]
```diff
@@ -50,6 +50,17 @@ The current version of media includes the following functionality:
 * Ability to embed tweets, Instagrams, and YouTube/Vimeo videos directly into
   CKEditor by pasting the video URL
 
+#### Extending Lightning Media (Contributed Modules)
+A number of contributed modules have been written by Drupal community to integrate Drupal Media Entity with Lightning. This allows a seamless integration between Lightning functionality and 3rd party media services via Media Entity. Below is a list of 3rd party services and contributed modules that are currently available for Lightning Media:
```
Comment (phenaproxima) [inline README.md]: Let's streamline this a little bit. How about:

"Drupal community members have contributed several modules which integrate Lightning Media with additional third-party media services. These modules are not packaged with Lightning or maintained by Acquia, but they are stable and you can use them in your Lightning site:"

File: README.md [hunk 7]
```diff
@@ -50,6 +50,17 @@ The current version of media includes the following functionality:
...
+
+  * [Facebook](https://www.drupal.org/project/lightning_media_facebook)
+  * [Imgur](https://www.drupal.org/project/lightning_media_imgur)
+  * [Flickr](https://www.drupal.org/project/lightning_media_flickr)
+  * [500px](https://www.drupal.org/project/lightning_media_d500px)
+  * [Soundcloud](https://www.drupal.org/project/lightning_media_soundcloud)
```
Comment (phenaproxima) [inline README.md]: Now I'm picking nits :) Can this be "SoundCloud", which is the actual name of the service? Note the capital "C".

**Review by phenaproxima — CHANGES_REQUESTED** [review]
This looks GREAT! Just two more changes and I am ready to merge it.

**Review by phenaproxima — APPROVED** [review]

**Comment by dakkusingh** [comment]
both done 👍

**Comment by phenaproxima** [comment]
I *could* wait for Travis to pass this, but honestly, it looks terrific and it's just documentation changes. I doubt this will fail any tests :) @balsama has OKed me to merge this.

**Pull request merged**